### PR TITLE
ReplacedTwistlockInstances

### PR DESCRIPTION
**Pull request by ssugandh**
Please review the "Twistlock" instances that I replaced with "Prisma Cloud Compute".

### CODE REVIEW

File: api/descriptions/settings/secrets_post.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ $ curl -k \
     {
       "name": "Cyberark",
       "type": "cyberark",
-      "appID": "Twistlock_Console",
+      "appID": "Prisma Cloud Compute_Console",
```
Comment (iansk) [inline api/descriptions/settings/secrets_post.md]: @ssugandh Not sure if spaces are acceptable here. Change to all underscores

**Review by iansk — CHANGES_REQUESTED** [review]

File: api/descriptions/settings/secrets_post.md (hunk 1 above)
Comment (ssugandh) [reply api/descriptions/settings/secrets_post.md]: @iansk I added the underscore. Please see my latest commit.

Comment (iansk) [reply api/descriptions/settings/secrets_post.md]: @ssugandh You also need to update the value in the example curl request.

**Review by iansk — APPROVED** [review]

**Comment by iansk** [comment]
Nice job, @ssugandh. I've just merged your first PR!

**Pull request merged**